tests/acceptance/AdminCest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Tests for core module's admin features.
+ *
+ * phpcs:disable WordPress.Files, WordPress.NamingConventions, PSR1.Classes.ClassDeclaration.MissingNamespace, HM.Functions.NamespacedFunctions
+ */
+
+use Codeception\Util\Locator;
+
+/**
+ * Test core module admin features.
+ */
+class AdminCest {
+
+	/**
+	 * Test module versions are displayed correctly on about page.
+	 *
+	 * @param AcceptanceTester $I Tester
+	 *
+	 * @return void
+	 */
+	public function testSupportTicketLink( AcceptanceTester $I ) {
+		$I->wantToTest( 'Support ticket link is correct.' );
+		$I->loginAsAdmin();
+		$I->amOnAdminPage( '/' );
+
+		$I->moveMouseOver( '.altis-logo-wrapper' );
+		$I->see( 'Open Support Ticket', Locator::href( 'https://dashboard.altis-dxp.com/#/support/new' ) );
+	}
+
+	/**
+	 * Test module versions are displayed correctly on about page.
+	 *
+	 * @param AcceptanceTester $I Tester
+	 *
+	 * @return void
+	 */
+	public function testSupportTicketLinkWithCloud( AcceptanceTester $I ) {
+		// Schedule running of _configSupportTicketLinkWithCloud in next Acceptance Test.
+		$rollback = $I->preLoadConfig( __FILE__, [ __CLASS__, '_configSupportTicketLinkWithCloud' ] );
```

Comment:
Do you know if each test method is a new session? Would it be possible sense to use the `_before()` and `_after()` methods for example? I guess that would affect every test in the file.

Reply:
I'm returning the function to be called by the method at the end, so it clears it after the work is done. But in case the method doesn't do that, it'll be cleared on shutdown.

We could in theory do a before and after, and use some naming convention similar to how data sources work in phpunit, but it seemed a bit excessive to do so. I guess we'll know how much we'll need that as write more tests.

FYI that we'll need something very similar to update composer config, unless we have a filter that could be used with `preloadConfig`, eg: to activate authorship, but I think that might be a bit late, will see.

Reply:
Hm yeah, was wondering about that. The config can be filtered and will start being used after the `altis.modules.init` action hook so I think this same approach could work.

This looks good, I'm just thinking through some ideas on the other PR that adds the `preLoadConfig` method.